WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,3 +1,6 @@
-- The Quick Start card and Next Steps tours match the new designs for existing sites.
-- Insert images from other sites using the “Insert from URL” option in the app.
-- Get more context through your screen reader with block editor accessibility improvements.
+* [*] Site creation: enhances the design selection screen with recommended designs [https://github.com/wordpress-mobile/WordPress-Android/pull/16468]
+* [*] DeepLinks: fix auto-verification of App Links in Android 12 [https://github.com/wordpress-mobile/WordPress-Android/pull/16696]
+* [*] Quick Start: Fix Quick Start focus points in RTL mode [https://github.com/wordpress-mobile/WordPress-Android/pull/16720]
+* [*] [internal] Block Editor: Bump react-native-gesture-handler to version 2.3.2. [https://github.com/wordpress-mobile/WordPress-Android/pull/16645]
+* [***] Jetpack App: Introducing blogging prompts. Build a writing habit and support creativity with a periodic prompt for inspiration. [https://github.com/wordpress-mobile/WordPress-Android/pull/16729]
```

Comment:
nit: to help our freelance contractor (who will wordsmith those notes into a nice editorialized copy) not being tempted to include the explicity include the mention of "Jetpack App" when she'll write the final copy for the Jetpack notes, I often remove that mention once our scripts have extracted the notes in those files.

TBH in practice that doesn't really matter much as she's used to it by now (just like she knows that she should not include items flagged as `[internal]` in the final user-facing wordsmith'd copy), so not strictly necessary to remove it by now… but eh 🤷‍♂️  doesn't hurt anyway 😛 

```suggestion
* [***] Introducing blogging prompts. Build a writing habit and support creativity with a periodic prompt for inspiration. [https://github.com/wordpress-mobile/WordPress-Android/pull/16729]
```